src/java.desktop/share/classes/sun/print/resources/serviceui_zh_CN.properties

```diff
@@ -63,7 +63,7 @@ label.pstype=类型:
 label.rangeto=至
 label.size=大小(&Z):
 label.source=来源(&C):
-label.outputbins=出纸托盘(&P):
+label.outputbins=输出托盘(&P)：
```

Comment:
Looks like the translation tool changed this from a colon (U+003A) to a full width colon (U+FF1A). There are l10n rules when it comes to punctuation, I am not sure if this is a result of that. It looks like the surrounding key/values are simply using (U+003A) colons. If we decide to revert this colon change, we need to also probably file a bug against the translation tool.

Reply:
Right, I agree. I'm OK with either decision too, but I'm leaning towards reverting this to standardize the colons since the rest are (U+003A). Maybe a reviewer from desktop can chime in once this PR is undrafted.